recipes-core/images/orangepi-image-initramfs.bb

```diff
@@ -0,0 +1,23 @@
+DESCRIPTION = "Small image capable of booting the orangepi R1+"
```

Comment:
Do we have to specify the initramfs in the image name?
I wonder if it's better to name it just name it `orangpi-core-image.bb` since this image won't work without it

Reply:
I believe initramfs is more clear, as it is meant to be compiled as an initramfs. It was inspired of core-image-minimal-initramfs.bb